cosmos-cmd-tlm-api/app/controllers/metadata_controller.rb

```diff
@@ -18,26 +18,25 @@
 # copyright holder
 
 require 'cosmos/models/metadata_model'
+require 'time'
 
 class MetadataController < ApplicationController
   def initialize
     @model_class = Cosmos::MetadataModel
   end
 
   def parse_time_input(x_start:, x_stop:)
```

Comment:
What is this?  Why does it default to +/- 7 days?

Reply:
I don't know but I didn't change it. It doesn't match the comment either which says start is  minus 12 hrs and end is plus 2 days. I'm honestly not sure when the search feature is used.